### DISCUSSION TITLE
Immature balance pplns

### DISCUSSION THREAD

**Discussion by totom9000** [coinfoundry/miningcore#1324] [Ideas]
Is there any chance  of getting imaturebalance  added  in pplns pool.

Like if a miner mined on a Block found i want to display  value aprox hes about to get from that Block while its maturing  and w8 on payout.

**Comment by totom9000** [comment]
its fixed we added support to this our self.

**Reply by MiningCryptoLive** [reply]
@totom9000 any chance you can share what needs to be changed to add this?
